lockfile/audit/src/index.ts

```diff
@@ -55,6 +62,57 @@ function getAuthHeaders (authHeaderValue: string | undefined) {
   return headers
 }
 
+async function fixFindingPaths (auditReport: AuditReport, opts: {
+  lockfile: Lockfile
+  lockfileDir: string
+  include?: { [dependenciesField in DependenciesField]: boolean }
+}): Promise<AuditReport> {
+  const { advisories } = auditReport
+  if (Object.keys(advisories).length) {
+    const prefixes = Object.keys(opts.lockfile.importers).map(importer => path.join(opts.lockfileDir, importer))
+    for (const { findings, module_name: moduleName } of Object.values(advisories)) {
+      for (const finding of findings) {
+        finding.paths = await searchPackagePaths([`${moduleName}@${finding.version}`], prefixes, {
+          lockfileDir: opts.lockfileDir,
+          depth: Infinity,
+          include: opts.include,
+        })
+      }
+    }
+  }
+  return auditReport
+}
+
+async function searchPackagePaths (packages: string[], projectPaths: string[], searchOpts: {
```

Comment:
Why do you use an array of packages if you always only pass one package?

Reply:
Thanks for your suggestion, there is no export for other places to use currently, it seems that there is really no need to use the array, I have updated it.